sakura_core/basis/CMySize.h

```diff
@@ -32,7 +32,7 @@
 class CMySize : public SIZE{
 public:
 	//コンストラクタ・デストラクタ
-	CMySize(){} //※初期化なし
+	CMySize() = default; //※初期化なし
```

Comment:
I think there is no problem with the fix itself.

However, I think the original code has something inappropriate, so it is preferable to make the following corrections.
```suggestion
	CMySize() : CMySize(0, 0) {}
```

Reply:
@berryzplus fix it https://github.com/sakura-editor/sakura/pull/1860/commits/1fc1801a340c0d1ebb2b306d27b3d48c14ec2c17